docs/en/latest/plugins/ai-content-moderation.md

```diff
@@ -0,0 +1,220 @@
+---
+title: ai-content-moderation
+keywords:
+  - Apache APISIX
+  - API Gateway
+  - Plugin
+  - ai-content-moderation
+description: This document contains information about the Apache APISIX ai-content-moderation Plugin.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `ai-content-moderation` plugin processes the request body to check for toxicity and rejects the request if it exceeds the configured threshold.
+
+**_This plugin must be used in routes that proxy requests to LLMs only._**
```

Comment:
Just routes? no services?

Or do you just want to stress the upstream should be LLM providers.

Reply:
> Or do you just want to stress the upstream should be LLM providers.

this.

Reply:
In that case I actually think this sentence is redundant. how about just mention `It is used when integrating with LLMs.` in the paragraph above?